Website/src/rendering/src/assets/css/components/Navigation/Breadcrumb.css

```diff
@@ -1,6 +1,10 @@
 .breadcrumb {
   @apply max-w-screen-2xl mx-auto overflow-hidden bg-transparent p-7 capitalize;
 
+  &.shop-breadcrumb {
```

Comment:
Could that class please be extracted into a `ShopBreadcrumb.css` file with the first line as `.breadcrumb.shop-breadcrumb {` ?
That would keep the code cleaner with one CSS file per component file.

Reply:
ShopBreadcrumb.css was created [here](https://github.com/Sitecore/Sitecore.Demo.Edge/pull/322/files#diff-bb2fbd9b0b5c7e7bacd2e716f23f82ab3d2ccf18befb0d8b9013379963062a19), maybe we should merge develop first?